### DISCUSSION TITLE
Access to Github Discussions API?

### DISCUSSION THREAD

**Discussion by fjania** [MeltanoLabs/tap-github#244] [Q&A]
I've looked through the code, and played with a sample github tap locally, and I don't see a way to access the Discussions API.

There is a [GraphQL API](https://docs.github.com/en/graphql/guides/using-the-graphql-api-for-discussions) for it, and I see that there's a `GitHubGraphqlStream` class that could be built off of in the existing codebase.

Before I look into contributing a Discussions stream implementation, I want to make sure I'm not missing something. Is there some other way that Discussions are handled that maybe I'm just missing?

**Comment by edgarrmondragon** [comment]
@fjania It does seem to be missing. PRs welcome!
